focus-areas/place/collaboration-platform-activity.md

```diff
@@ -32,26 +39,37 @@ The unit of data collection is the individual activity on a platform. Metadata r
 * [Contribution attributions](https://chaoss.community/metric-contribution-attribution/) (e.g., people or organizations)
 
 ### Visualizations
+1. **Visualization of Platform Activity:**  
+   Display the count of platform activities over time.
+   ![GrimoireLab Implementation](https://raw.githubusercontent.com/chaoss/wg-metrics-development/main/focus-areas/place/images/collaboration-platforms.png)  
+   *Figure 1: Collaboration platform activity chart (GrimoireLab)*
 
-![GrimoireLab Implementation](https://raw.githubusercontent.com/chaoss/wg-common/main/focus-areas/place/images/collaboration-platforms.png)
-
-https://chaoss.biterg.io/app/kibana#/dashboard/ab68fe20-17f2-11e9-872f-e17019e68d6d
+2. **Interactive Dashboards:**  
+   [GrimoireLab Dashboard](https://chaoss.biterg.io/app/kibana#/dashboard/ab68fe20-17f2-11e9-872f-e17019e68d6d)
+   *Figure 2: GrimoireLab dashboard displaying platform activity metrics (GrimoireLab)*
 
-### Tools Providing the Metric
-* Orbit.love is a community management platform that captures this for a few channels: https://docs.orbit.love/docs/adding-activities
-* GrimoireLab
-* Augur
+</details></span>
 
 ## References
-* Related metric: https://chaoss.community/metric-chat-platform-inclusivity/
-* Related metric: https://chaoss.community/metric-issues-new/
+- [Related Metric: Chat Platform Inclusivity](https://chaoss.community/metric-chat-platform-inclusivity/)
+- [Related Metric: Issues New](https://chaoss.community/metric-issues-new/)
```

Comment:
Same here with the URLs:
- Chat Platform Inclusivity: https://chaoss.community/?p=3536
- Issues new: https://chaoss.community/?p=3634